templates/base.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  {% load static %}
+  <link rel="stylesheet" href="{% static 'css/styles.css' %}">
+  {% block title %}
+    <title>Taxi Service</title>
+  {% endblock %}
+</head>
+<body>
+  {% block sidebar %}
+    {% include "includes/sidebar.html" %}
+  {% endblock %}
```

Comment:
are u sure that sidebar should be at the same level as content block? 
move ur sidebar to the left/right side of page

Reply:
i separated it from content
![image](https://user-images.githubusercontent.com/72259870/226868725-76ad806f-9d97-4a8f-bb2a-df19be498140.png)